pkg/controllers/disruption/consolidation.go

```diff
@@ -89,6 +93,7 @@ func (c *consolidation) markConsolidated() {
 
 // ShouldDisrupt is a predicate used to filter candidates
 func (c *consolidation) ShouldDisrupt(_ context.Context, cn *Candidate) bool {
+	// TODO: Remove the check for do-not-consolidate at v1
```

Comment:
Do you need this TODO? Can we cut an issue instead?

Reply:
@jonathan-innis and I have been using TODOs to drive some of the deprecation work, but I think this could be included in #758. Sound good to you?

Reply:
I'd like to see the TODOs in code _and_ it captured in an issue. This makes it easier to understand what should be removed and why it can be removed later.